sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -65,6 +65,7 @@
 - Continuation token of result types are changed to nullable.
 - Rename `Models::DeleteSnapshotsOption::Only` to `Models::DeleteSnapshotsOption::OnlySnapshots`.
 - Rename `SourceConditions` in API options to `SourceAccessConditions`.
+- Remove Blob Batch.
```

Comment:
Out of curiosity, what motivated the removal?

Reply:
It will come back as a separate package as suggested by Ted.